test/config/tenants.yaml

```diff
@@ -27,10 +27,9 @@ tenants:
   opa:
     url: http://127.0.0.1:8181/v1/data/observatorium/allow
   rateLimits:
-    "/api/metrics/v1/":
+    "/api/metrics/v1/.+/api/v1/receive":
```

Comment:
In general, it makes things messy for configuration files to have maps where the keys are special values that users can define. I think it will be cleaner and easier to understand if we use a list:
```yaml
rateLimits:
- path: /api/metrics/......
  limit: 1
  window: 1s
- path: /api/logs/.......
  limit: 100
  window: 1s
```